docs/configuration.md

```diff
@@ -303,4 +308,22 @@ prod config:
 {'my_project': 'I_am_prod'}
 dev config:
 {'my_project': 'I_am_dev'}
+```
+
+## Deployment config
+
+`DeploymentConfig` is a subclass of the `Config` class with one additional, optional argument – `environment_variables_prefix`.
+Contrary to the `Config` class, the `DeploymentConfig` allows you to set a custom environment variables name prefix (overriding the default `bf_` value).
+It's useful in CI/CD systems like Bamboo.
```

Comment:
CI/CD servers

Reply:
btw why changing this prefix would be useful on CI/CD server?

Reply:
done